pkg/datapath/linux/node.go

```diff
@@ -897,6 +899,25 @@ func (n *linuxNodeHandler) nodeUpdate(oldNode, newNode *nodeTypes.Node, firstAdd
 		oldKey = oldNode.EncryptionKey
 	}
 
+	if !n.nodeConfig.EnableIPSec {
+		bpfIPCacheList := make(map[string][]string)
+		if err := ipcache.IPCache.Dump(bpfIPCacheList); err != nil {
+			// TODO: Write error
+		}
```

Comment:
To check values from IPCache I've used IPCache.Dump() from github.com/cilium/cilium/pkg/maps/ipcache . Would this be the correct functions to use?